src/test/java/integration/EntriesApiTest.java

```diff
@@ -311,6 +315,25 @@ void getDynamicFieldsEntry_ReturnDynamicFields() {
     }
 
     @Test
+    void getEntryByFullPath_ReturnRootFolder() {
+        FindEntryResult entry = repositoryApiClient.getEntriesClient().getEntryByPath(repoId, rootPath, false).join();
+        assertNotNull(entry);
+        assertTrue(entry.getEntry().getId() == 1);
+        assertTrue(entry.getEntry().getFullPath().equals(rootPath));
+        assertTrue(entry.getEntry().getEntryType().toString() == "Folder");
```

Comment:
In Java, it's highly recommended to not compare strings with ==. use equals or equalsIgnoreCase methods